libs/designsystem/accordion/src/accordion.directive.ts

```diff
@@ -1,8 +1,21 @@
-import { Directive } from '@angular/core';
+import { AfterContentInit, ContentChildren, Directive, Input, QueryList } from '@angular/core';
+import { AccordionItemComponent } from './accordion-item.component'; // Adjust path as needed
 
 @Directive({
   selector: '[kirbyAccordion], kirby-accordion',
   exportAs: 'kirby-accordion',
   standalone: false,
 })
-export class AccordionDirective {}
+export class AccordionDirective implements AfterContentInit {
+  @Input() headingLevel: number;
+  @Input() role: string;
+  @ContentChildren(AccordionItemComponent) listChildren: QueryList<AccordionItemComponent>;
```

Comment:
Since we are in accordion and not the list here, I think the naming could be improved. Consider something like `accordionItems` or `accordionChildren`